src/platform/windows.rs

```diff
@@ -356,6 +356,9 @@ pub trait WindowExtWindows {
     unsafe fn window_handle_any_thread(
         &self,
     ) -> Result<rwh_06::WindowHandle<'_>, rwh_06::HandleError>;
+
+    /// Set whether a mouse click activates the window.
```

Comment:
```suggestion
    /// Set whether a mouse click activates the window.
    ///
    /// Defaults to `true`.
```
Same at `WindowAttributesExtWindows`.